README.md

```diff
@@ -41,6 +41,23 @@ $ scalingo run "gm version"
 
 ## Hacking
 
+### New way
+
+You could upload the desired version of GraphicMagick from [GraphicMagick sources](https://hg.osdn.net/view/graphicsmagick/GM/tags)
+on a S3 for example. Then using the environment variable `ARCHIVE_URL` to specify
+the archive location, following the example:
+
+```shell
+scalingo env-set ARCHIVE_URL=https://hg.osdn.net/view/graphicsmagick/GM/archive/GraphicsMagick-1_3_36.tar.gz
+```
+
+> Note that Scalingo provide only support of the default version defined in `configs.sh` file.
```

Comment:
`only support of default the default`. Hmm, a more understandable sentence please?? 😄